modules/catch2/3.3.0/source.json

```diff
@@ -0,0 +1,9 @@
+{
+    "url": "https://github.com/catchorg/Catch2/archive/refs/tags/v3.3.0.tar.gz",
+    "integrity": "sha256-/i8ppUyndcLdBLuX/7edOY5iEOPKoXQ0i1zTt+TKiH0=",
+    "strip_prefix": "Catch2-3.3.0",
+    "patch_strip": 0,
+    "patches": {
+        "module_dot_bazel.patch": "sha256-0ZKA9VjsNb7SM6kYVsfPs+XBKKFVGTYgnHFpohIOkXA="
```

Comment:
This integrity value has to be updated as the module file patch has changed.

Reply:
Updated it. Thank you for pointing it out.